monai/transforms/post/array.py

```diff
@@ -100,7 +105,7 @@ def __call__(
         if sigmoid or self.sigmoid:
             img_t = torch.sigmoid(img_t)
         if softmax or self.softmax:
-            img_t = torch.softmax(img_t, dim=0)
+            img_t = torch.softmax(img_t, dim=self.kwargs.get("dim", 0))
```

Comment:
Here actually we didn't use other args except for `dim`.
How about also adding `kwargs` to the `call()` function, and change the overall logic to:
```py
def __call__(..., **kwargs):
    ...
    kwargs = self.kwargs if kwargs is None else kwargs
    if "dim" not in kwargs:
        kwargs["dim"] = 0
    ...
    img_t = torch.softmax(img_t, **kwargs)
```

Thanks.

Reply:
For the record it'll be useful for being robust to deprecated args.